proto/klyraprotocol/blocktime/query.proto

```
@@ -6,37 +6,16 @@ option go_package = "github.com/StreamFinance-Protocol/stream-chain/protocol/x/b
 import "gogoproto/gogo.proto";
```

Buf (1.47.2)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)